deploy.php

```diff
@@ -375,7 +375,16 @@
 			$headers = array();
 			$headers[] = sprintf('From: Simple PHP Git deploy script <simple-php-git-deploy@%s>', $_SERVER['HTTP_HOST']);
 			$headers[] = sprintf('X-Mailer: PHP/%s', phpversion());
-			mail(EMAIL_ON_ERROR, $error, strip_tags(trim($output)), implode("\r\n", $headers));
+
+			if (is_string(EMAIL_ON_ERROR)) {
+				$emails = array(EMAIL_ON_ERROR);
+			} elseif (is_array()) {
```

Comment:
`is_array()` requires a variable that's being checked

    bool is_array ( mixed $var )